docs/contracts/abstract-contracts.rst

```diff
@@ -52,3 +52,8 @@ facilitating patterns like the `Template method <https://en.wikipedia.org/wiki/T
 Abstract contracts are useful in the same way that defining methods
 in an interface is useful. It is a way for the designer of the
 abstract contract to say "any child of mine must implement this method".
+
+.. note::
+
+  Abstract contracts cannot override an implemented virtual function with an
```

Comment:
No contract is able to do that, right?

Reply:
Yes (in the sense that override checker disallows it) But the context is specific for abstract contracts.

Reply:
LGTM.

> No contract is able to do that, right?

I think the original issue was created because non-abstract contracts by definition cannot leave any inherited unimplemented function unimplemented but abstract contracts can with the exception of previously implemented functions that they override.